WH's NEW MORE RANDOM ROOMS

WonderPsycho opened the pull request:
# About The Pull Request
does what the title says on the PR... adds new Random Rooms to Bee Maint Generation, plenty of new, !!FUN!!, and quite rare new random rooms in maint, the more rarer ones to generate being quite dangerous with what it contains so I made those rooms super rare specifically at "weight = 1", my main inspiration to do this PR was to make maint exploration in Monke more fun and diverse and i have plans on making more random maint rooms in the future (now personally I don't think this PR will possibly be merged but atleast I tried my best on adding some new random rooms to Monke code's Bee randomly generated maint system) 


## Why It's Good For The Game

makes maint exploring more fun and diverse to enjoy, and some challenges to find when exploring the many dark abandoned maintenance tunnels of a NanoTrasen Space Station possibly containing old abandoned failed NanoTrasen experiments, or some space hobo den where some space hobos make some drugs.
The PR also adds new random room template .dmm files to make it easier for those who want to make their own new random rooms for Monke's version of Bee's Random Room Randomly generated maint system!

images of the rooms:https://imgur.com/gallery/GVfiUIF

Changelog
🆑
add:adds in more rooms for monke code's random maint, all mapped out by WonderHellblazer/WonderPsycho, also adds some blank templates to help if anyone else wants to make random rooms of their own for monke code
/🆑

Merlin1230 commented:
Could we get a picture of all the rooms?

WonderPsycho commented:
> Could we get a picture of all the rooms?
i don't know how to add images in PRs and comments but i'll try to add images of all of the rooms

WonderPsycho commented:
> Could we get a picture of all the rooms?

aight i managed to figure something out on adding images, so i edited my PR comment to link to a imgur of all the screenshots i took of the maps and all the screenshots i took recently of the maps, it's still a bit in the works gotta screenshot some other smaller rooms i made for this PR

WonderPsycho commented:
> > Could we get a picture of all the rooms?
> 
> aight i managed to figure something out on adding images, so i edited my PR comment to link to a imgur of all the screenshots i took of the maps and all the screenshots i took recently of the maps, it's still a bit in the works gotta screenshot some other smaller rooms i made for this PR

aight i have screenshotted all the random rooms i made, and added them to the imgur link and stuff so yeah

WonderPsycho commented:
again a reminder that the more chaos causing and more dangerous rooms or rooms that contain the more dangerous stuff are the most rare as i have them set to 1 on the weight for them so yeah

Merlin1230 commented:
okay, I have other comments for the other rooms, but I'll start out with this one.
![image](https://user-images.githubusercontent.com/76177064/155765111-c4143314-cde2-4999-9f68-c6412a03b288.png)
First off, the ids with sec access should not be there. Remember, this is in maints where some staffie can walk in and now they have access to all of sec. The ghost roles should just spawn with sec IDs.
I'm also very iffy on the idea of having all of this sec gear... locked behind maint access.

WonderPsycho commented:
> okay, I have other comments for the other rooms, but I'll start out with this one. ![image](https://user-images.githubusercontent.com/76177064/155765111-c4143314-cde2-4999-9f68-c6412a03b288.png) First off, the ids with sec access should not be there. Remember, this is in maints where some staffie can walk in and now they have access to all of sec. The ghost roles should just spawn with sec IDs. I'm also very iffy on the idea of having all of this sec gear... locked behind maint access.

well i am sorry but i am unable to give access to the ghost role players cause for some reason it doesn't let me, i tried giving access to the ghost roles but it doesn't work, the Charlie Station ids still doesn't have access to anything in the station, i also considered putting the ids in crates but at the same time i feel the ghost role seccies wouldn't check the crates for the access out of the room

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/skwh_rdm139
+	name = "SAW GBS Maint Game Room"
+	room_id = "skwh_rdm139_GBSsawgameroom"
```
This room is extremely imbalanced and can be cheesed by simply reading the descriptions of each bottle.
GBS is an admin-only virus.

WonderPsycho replied:
eh i'll remove this map, i originally intended this map to be a funny SAW reference room anyways but figured it was kind of gonna be imbalanced

Merlin1230 commented:
I haven't looked at how ghost roles work, but you probably need to make a custom one for this room.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
```
Contains far too many disease samples AND Fungal TB.
TB is a nuke op only disease that is stronger than 90% of player created diseases.

WonderPsycho replied:
will revamp alot to be less too much of it as it is, plus while removing the Fungal TB, i should also remove the "WIZARD CONTAGION" bottle, that bottle is Wizarditis and i believe bottle might be a bit too much maybe?

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
```
Besides the overabundance of virus samples, this isn't terribly imbalanced.

WonderPsycho replied:
aight i'll replace all the blood in this room with fresh blood as i have many more maps to focus on in putting in more dresh blood then dried blood, plus i wanna help fix this PR real quick before i get back into working on some other stuff i want to work on so yeah

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/skwh_rdm159
+	name = "Monkey Testing Room"
+	room_id = "skwh_rdm159_MONKETESTSUBJECT"
```
Contains enough samples to get a single virologist every single disease possible.
Contains GBS, Brainrot and Fungal TB, inaccessible diseases for crew.
Contains unused Experimental Viral Base, which was removed from virology after the Extrapolator was introduced.

WonderPsycho replied:
eh i'll remove this map as now looking back at it, i kind of just don't like this map anymore, and in my opinion kind of isn't that much of a fun room to find so yeah

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/skwh_rdm160
+	name = "Green Viney Riches"
+	room_id = "skwh_rdm160_greenvineyriches"
```
Contains far too many virus samples.
The space cash there has 0 value.
Contains security IDs and antagonist gear.
That many kudzu seeds are going to lead to issues.

WonderPsycho replied:
huh i thought i put more fresh blood then dried blood in this room, and i'll remove the dead bodies that have the sec ids and antagonist gear

WonderPsycho replied:
also just question when you mean that space cash has 0 value, do you mean it should spawn with 0 value or does it just spawn with 0 value? cause in the paper scrap i put in the room with what i wrote in it, i intended the space cash in the room to have 0 value but forgot to change many of the space cash/credits to 0 value, cause i intended the money to be "counterfitted nanotrasen credits" or something like that, sorry for being a bit confused by what you mean by "The space cash has 0 value"

WonderPsycho replied:
wait no now i know what you meant you mean the regular space cash has 0 value aight removing those sorry for the confusion

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/maskroom3x5
+	name = "Haunted Old Maskroom"
+	room_id = "maskroom3x5"
```
Besides the 100% chance Cluwne mask, this is fine.

WonderPsycho replied:
aight removing the cluwne mask from this room

WonderPsycho replied:
besides bee code uses tg cluwne anyways

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+
+/datum/map_template/random_room/secretivemaintwater
+	name = "Secretive Maint Water Collector Room 1"
+	room_id = "secretivemaintwatercollectorroom"
```
I'd tone it down to maybe 2-3 bottles of random reagents, but it seems okay otherwise.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+	weight = 4
+	stock = 6
+
+/datum/map_template/random_room/DRUGDEALERHOBOS
```
This needs to be in lowercase.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/DRUGDEALERHOBOS
+	name = "Hobo Drug Production Room"
+	room_id = "DRUGDEALERHOBOS"
+	mappath = "_maps/RandomRooms/10x5/DRUGDEALERHOBOS.dmm"
```
Too many virus samples.
Contains Eldritch Essence, a cult chem that is far too powerful to leave around.
Contains Beesease, an inaccessible disease.
Two full chemical dispensers in maint is too imbalanced.

Merlin1230 commented:
Also, another thing, please put these maps into the monkestation folder.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/randomlostartifact
+	name = "Room of the Lost Maint Artifact of OLD"
+	room_id = "randomlostartifact"
+	mappath = "_maps/RandomRooms/5x3/randomlostartifact.dmm"
```
This room is a bit bland, but acceptable.
A single rare artifact isn't going to cause too much issue.

WonderPsycho commented:
> Also, another thing, please put these maps into the monkestation folder.

aight will do

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+	weight = 3
+	stock = 2
+
+/datum/map_template/random_room/ARTROOM
```
This needs to be lowercase.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/ARTROOM
+	name = "Hazardous Abandoned Artifact Room"
+	room_id = "ARTIFACTROOM"
+	mappath = "_maps/RandomRooms/10x5/ARTIFACTROOM.dmm"
```
No.
This many artifacts will give full end-game research unlocks instantly.
Tritium isn't even a threat.

WonderPsycho replied:
will have to say the tritium in the screenshot was actually caused by one of many random artifacts in the room, i did not put tritium in the room manually infact the artifacts that do some weird random shit in the room if they were generated with being able to do stuff if in the darkness, and in some cases the artifact sometimes floods the room with other random gasses like plasma and nitrogen so yeah, just something to note, but yeah the room is removed from the PR

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/BlobInfestedXenobio
+	name = "Abandoned Old Blob Infested Xenobio Room"
+	room_id = "XenobioBlobInfested"
+	mappath = "_maps/RandomRooms/10x10/XenobioBlobInfested.dmm"
```
No.
Contains ghost roles.
If you had actually tested this room in-game, you can see the blob actually can spread outside of the room instantly.

WonderPsycho replied:
will say i have tested this room many times and had been trying to reinforce the walls to stronger walls to keep the blob node stuff in, also the ghost role was not intended, but got turned into a ghost role cause one of the blob spores keep taking over the dead corpse in the room, anyways i deleted this room from the PR

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+	template_width = 10
+	weight = 1
+
+/datum/map_template/random_room/SECONDAI
```
Needs to be lowercase.

WonderPsycho commented:
> > Also, another thing, please put these maps into the monkestation folder.
> 
> aight will do


Will note this is gonna take me a while as i also have to re-edit the randomrooms.dm file or make a copy of it and rearrange it to get the new locations of the map files and stuff

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/SECONDAI
+	name = "Holy Hobo AI Worship Room"
+	room_id = "SECONDARYAIWORSHIPROOM10x10"
+	mappath = "_maps/RandomRooms/10x10/SECONDARYAIWORSHIPROOM10x10.dmm"
```
The AI does not actually spawn in.
As it is, it seems to have fine balance.
But the AI actually spawning in would make it unacceptable.

WonderPsycho replied:
aight will change it to a AI frame if that's fine since there is another maint room that did the whole abandoned AI frame thing

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+	weight = 2
+	stock = 1
+
+/datum/map_template/random_room/MAINTSEC
```
This needs to be lowercase.

Merlin1230 commented:
> Will note this is gonna take me a while as i also have to re-edit the randomrooms.dm file or make a copy of it and rearrange it to get the new locations of the map files and stuff

Also, I would put all of the new rooms in a randomrooms.dm located.. somewhere in the monkestation folder. You should really only need to change the path on the datums as well.

WonderPsycho commented:
> > Will note this is gonna take me a while as i also have to re-edit the randomrooms.dm file or make a copy of it and rearrange it to get the new locations of the map files and stuff
> 
> Also, I would put all of the new rooms in a randomrooms.dm located.. somewhere in the monkestation folder. You should really only need to change the path on the datums as well.

yeah that is what i was saying earlier

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/MAINTSEC
+	name = "Maintenance Security Patrol Room"
+	room_id = "MaintSecurityPatrol"
+	mappath = "_maps/RandomRooms/10x5/MaintSecurityPatrol.dmm"
```
Absolutely hard no.
Security ghost roles will lead to metagrudging in the first place.
The crates literally contain more deadly weaponry than the armoury itself.
The banners are the non-decorative kind that actually give buffs to players.

WonderPsycho replied:
didn't realize the banners gave buffs yikes, i was intending to add in the decorative banner must of confused it with another file but yeah i'll get rid of the maps  you don't want and reedit some others you allow

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+	weight = 5
+	stock = 4
+
+/datum/map_template/random_room/HOBOJERRY
```
This needs to be lowercase.

nednaZ commented on code/modules/mapping/random_rooms.dm:
```diff
@@ -1479,3 +1479,139 @@
+/datum/map_template/random_room/HOBOJERRY
+	name = "Hobo Jerry's Room"
+	room_id = "HoboJerrysRoom"
+	mappath = "_maps/RandomRooms/5x4/HoboJerrysRoom.dmm"
```
Hard no.
Contains spellbooks for Knock, Mindswap and Smoke.
Contains a manual for actual CQC, a nuke op only item.
Too many random reagent water bottles.

WonderPsycho commented:
aight i did it, i moved all the files to the monkestation folder and made a new random_room.dm file for the monkestation _maps folder stuff

WonderPsycho commented:
aight Zanden so far the stuff you issued me on rebalancing and removing have all been done and i hope you enjoyed the new rebalanced stuff, fixed stuff and such

nednaZ commented:
I'll do a second review pass once it passes the unit tests.

WonderPsycho commented:
> I'll do a second review pass once it passes the unit tests.

aight

WonderPsycho commented:
> > i am still confused why this PR haven't passed one of the run tests
> 
> read the linter. ![image](https://user-images.githubusercontent.com/41448081/156302334-aedf66aa-3422-4df7-b820-a2f85c7b47ce.png)

ah it failed cause it considered "random_rooms.dm" to be missing? i actually just made a new random_rooms.dm file in the monkestation folder cause i was told to, and got rid of all of the new rooms listed in the original random_rooms.dm file and moved them to the monkestation random_rooms.dm file, i think maybe it doesn't work like that and have the the rooms relisted on that file with "//Monkestation Edit" or something like that

WonderPsycho commented:
i have a feeling this one failed run test is what's holding back on the re-review of this PR, i honestly just want this PR finished and over it so it can be either closed or merged

nednaZ commented:
Your issue is that you need to add a new line to the 'random_rooms.dm' file.

Merlin1230 commented:
>I have a feeling this one failed run test

Not even a run test, its just angy that you don't have a empty line at the end of the `random_rooms.dm` file.

Trust me, this "error" is definitely not as bad as what you COULD be getting. It should take about 2 seconds to fix.

WonderPsycho commented:
> > I have a feeling this one failed run test
> 
> Not even a run test, its just angy that you don't have a empty line at the end of the `random_rooms.dm` file.
> 
> Trust me, this "error" is definitely not as bad as what you COULD be getting. It should take about 2 seconds to fix.

oh hah, well i'll fix that right now real quick, well that's silly

WonderPsycho commented:
and i think i fixed it tho i need to look into it a little more

WonderPsycho commented:
> and i think i fixed it tho i need to look into it a little more

nevermind no i didn't fixed it, it's still giving out this weird run test error it keeps saying it's "missing" a line

WonderPsycho commented:
> > and i think i fixed it tho i need to look into it a little more
> 
> nevermind no i didn't fixed it, it's still giving out this weird run test error it keeps saying it's "missing" a line

cause by far i can't find where is this causing this weird line error

WonderPsycho commented:
wait a sec i think i found it

WonderPsycho commented:
ok this error is making no sense at all for me as to why thi WAIT WAHT TWO ERRORS NOW?

WonderPsycho commented:
yeah i don't get this error, it seems more like it's trying to look for the new lines with the random room maps i made but removed and moved over into a new random_rooms.dm file in the monkestation folder, btw fixed up the other errors as i realized my mistake earlier

nednaZ commented:
It's looking for a new line at the very end of the file.
When you press enter in this window, that's a new line.

All you have to do is go into that file and hit enter at the very end so you have a new line at the very bottom.

WonderPsycho commented:
> It's looking for a new line at the very end of the file. When you press enter in this window, that's a new line.
> 
> All you have to do is go into that file and hit enter at the very end so you have a new line at the very bottom.

ah makes sense now

WonderPsycho commented:
AAAnnnddd fixed

nednaZ commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,85 @@
+/datum/map_template/random_room
```
This section needs to go.
All of this is already defined in the main file itself, doesn't need a second, identical, definition

nednaZ reviewed (commented):
The entire set of maps currently cannot load as monkestation/code/modules/mapping/random_rooms.dm is not defined within beestation.dme

nednaZ commented:
I do have to request that you test and confirm that your changes actually work before requesting review.

WonderPsycho commented:
again i was told to make the second random_rooms.dm file in the monkestation folder, figured this wasn't gonna work, i am gonna fix this up rn add the rooms back to the random_rooms.dm file as a "//Monkestation Edit" and test it after my breakfast

nednaZ commented:
It will work if you add it to the .dme file, as with every other file inside.

WonderPsycho commented:
alright then

WonderPsycho commented:
and i fixed it and added the random_rooms.dm file from the monkestation folder into the dme file and tested it and it works now

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+/datum/map_template/random_room/skwh_rdm158
```
can you make the typepaths a little more descriptive of what the room is?

Also, I feel like the prefix (`skwh_rdm`) for all of the random rooms is unnecessary

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	weight = 2
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
```
same for here

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	template_width = 10
+	weight = 3
+
+/datum/map_template/random_room/skwh_rdm160
```
anddddd here

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "monkestation/_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
```
To me this feels like a LOT of cultures, I dont know what the non-random diseases do off the top of my head but they sound.. fairly bad. Also, the room does NOT need a chem dispenser, viro doesn't even get one.

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
+	mappath = "monkestation/_maps/RandomRooms/10x5/skwh_rdm157_slimestorage.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 10
+	weight = 3
```
I dont know WHY this room needs a kudzu seed, honestly.

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	weight = 3
+
+/datum/map_template/random_room/skwh_rdm160
+	name = "Green Viney Riches"
+	room_id = "skwh_rdm160_greenvineyriches"
+	mappath = "monkestation/_maps/RandomRooms/10x10/skwh_rdm160_greenvineyriches.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 6
```
this is WAY too many kudzu seeds, especially for a _weight six_ room.

WonderPsycho replied:
weird for some reason i remember giving this room a weight 2 or 1 but put 6 by accident, by this point i think i might just remove the room due to the amount of kudzu seeds, i can understand having the kudzu seeds of the slime storage removed, i mean i mainly just added those seeds to that room as just some random garbage players can get on the floor, but yeah

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	weight = 6
+
+/datum/map_template/random_room/maskroom3x5
+	name = "Haunted Old Maskroom"
+	room_id = "maskroom3x5"
+	mappath = "monkestation/_maps/RandomRooms/3x5/maskroom3x5.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 3
+	weight = 5
```
this one seems fine.

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	weight = 5
+
+/datum/map_template/random_room/secretivemaintwater
+	name = "Secretive Maint Water Collector Room 1"
+	room_id = "secretivemaintwatercollectorroom"
+	mappath = "monkestation/_maps/RandomRooms/3x3/secretivemaintwatercollectorroom.dmm"
+	centerspawner = FALSE
+	template_height = 3
+	template_width = 3
+	weight = 4
+	stock = 6
```
also fine.

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	stock = 2
+
+/datum/map_template/random_room/aigod
+	name = "Holy Hobo AI Worship Room"
+	room_id = "SECONDARYAIWORSHIPROOM10x10"
+	mappath = "monkestation/_maps/RandomRooms/10x10/SECONDARYAIWORSHIPROOM10x10.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 1
```
this seems fine, the combustible lemons are very out of place however.

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	stock = 5
+
+/datum/map_template/random_room/randomlostartifact
+	name = "Room of the Lost Maint Artifact of OLD"
+	room_id = "randomlostartifact"
+	mappath = "monkestation/_maps/RandomRooms/5x3/randomlostartifact.dmm"
+	centerspawner = FALSE
+	template_height = 3
+	template_width = 5
+	weight = 3
+	stock = 2
```
this one, is also fine

Merlin1230 commented on monkestation/_maps/random_rooms.dm:
```diff
@@ -0,0 +1,76 @@
+	stock = 6
+
+/datum/map_template/random_room/drugdealerhobos
+	name = "Hobo Drug Production Room"
+	room_id = "DRUGDEALERHOBOS"
+	mappath = "monkestation/_maps/RandomRooms/10x5/DRUGDEALERHOBOS.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 10
+	weight = 3
+	stock = 5
```
Why does a _hobo_ chem room need medical boards?

Merlin1230 reviewed (changes requested):
Left comments for all of the rooms.

WonderPsycho commented:
anyways will say i will try my best on getting these more rebalancing changes done later, i need to have myself some breakfast first and stuff, and honestly i just want to get this done whether or not it will just be closed and not merged or will be merged, kind of want to focus on other things i am working on then focus on this one too much, hence why i am kind of considering removing Green Viney Riches as well

WonderPsycho commented:
and again i originally thought i gave green viney riches a 1 or 2 in weight, that was a mistake on my part as it seems i gave the wrong weight size on how it will be generated to a 6 then what i originally intended

WonderPsycho commented:
actually i might just removed Large Abandoned Viro and Greenv Viney Riches entirely, out, both are too imbalanced especially the green viney riches on imo, cause it has too much money people can use to get pretty goddamn rich and shit

WonderPsycho commented:
btw i finished the new rebalancing changes and removed Large Viro and  Green Viney Riches, also removed the kudzu seeds of the slime storage room, and removed the conbustable lemons of the abandoned AI room with more Hobos, and removed the randomly spawned medical circuit boards sorry for making those really imbalanced rooms anyways i wasbasically just trying to get the hang of mapping and part of my mindset with ss13 mapping is because for most of my life on ss13 i personally view ss13 as more based on specticle on how chaotic and fun a mix between roleplay and chaos can get and such, next time i'll make more random rooms i'll focus far more on balance, plus i have plans on making a new rebalanced Hobo Jerry room this time not severely imbalanced as it originally was and some other stuff,

WonderPsycho commented:
> Left comments for all of the rooms.

just to let you know i finished doing the requested changes

WonderPsycho commented:
awaiting for Merlin to notice i have finished the changes they have requested

WonderPsycho commented:
readded the changelog, realizing i was constantly removing the changelog off of the PR without realizing it

nednaZ reviewed (approved):
Balance looks good.

Merlin1230 reviewed (approved):
It all looks good to me
